src/Core/Plugin.php

```diff
@@ -285,18 +285,27 @@ public static function bootstrap($name)
     }
 
     /**
-     * Returns true if the plugin $plugin is already loaded
-     * If plugin is null, it will return a list of all loaded plugins
+     * Returns true if the plugin $plugin is already loaded.
      *
-     * @param string|null $plugin Plugin name.
-     * @return bool|array Boolean true if $plugin is already loaded.
-     *   If $plugin is null, returns a list of plugins that have been loaded
+     * @param string $plugin Plugin name.
+     * @return bool
      */
-    public static function loaded($plugin = null)
+    public static function isLoaded(string $plugin): bool
```

Comment:
These look like methods we should also backport to 3.next so that we can add deprecation warnings in 3.7.